From 103f2129530c1c4d4a1a99419005dcdd08d38748 Mon Sep 17 00:00:00 2001
From: Adam Williamson <awilliam@redhat.com>
Date: Wed, 26 Jan 2022 15:44:31 -0800
Subject: [PATCH 2/2] Drop "CI" test setup that needs catchsegv

glibc has dropped catchsegv:
https://sourceware.org/bugzilla/show_bug.cgi?id=14913
so mutter no longer builds with recent glibc.

Signed-off-by: Adam Williamson <awilliam@redhat.com>
---
 meson.build | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/meson.build b/meson.build
index 4f7f023043..2a963b3b37 100644
--- a/meson.build
+++ b/meson.build
@@ -327,18 +327,6 @@ if have_tests
   )
 
   add_test_setup('plain')
-
-  add_test_setup('CI',
-    env: [
-      'MUTTER_DEBUG_DUMMY_MODE_SPECS=800x600@10.0',
-    ],
-    exe_wrapper: [
-      default_test_wrappers,
-      find_program('catchsegv'),
-      find_program('xvfb-run'), '-a', '-s', '+iglx -noreset',
-    ],
-    timeout_multiplier: 10,
-  )
 endif
 
 have_profiler = get_option('profiler')
-- 
2.35.0.rc1

